### DISCUSSION TITLE
Update example at https://flet.dev/docs/reference/types/window#destroy

### DISCUSSION THREAD

**Discussion by Razielthc** [flet-dev/flet#3813] [Ideas]
Please update the example of the destroy method in Update example at https://flet.dev/docs/reference/types/window#destroy the example is confusing since in the line page.window.on_event = window_event the reference window_event does not exist It should be event that calls the function with the same name or update the function name with the reference window_event
![image](https://github.com/user-attachments/assets/43144297-a02b-4494-9ef2-58f8ff1d1335)

**Comment by ndonkoHenri** [comment]
Will do it, thanks!
